src/Frontend/src/views/throughputreport/throughputClient.ts

```diff
@@ -34,9 +34,32 @@ class ThroughputClient {
     return data;
   }
 
-  public downloadReport(masks: string[]) {
-    //useDownloadFile();
-    //const response = await useFetchFromServiceControl(`${this.basePath}/report/file`);
+  public async downloadReport(spVersion: string) {
+    const response = await useFetchFromServiceControl(`${this.basePath}/report/file?${spVersion}`);
+    if (response.status == 200) {
+      var fileName = "throughput-report.json";
+      const contentDisposition = response.headers.get("Content-Disposition");
+      if (contentDisposition != null) {
+        try {
+          fileName = contentDisposition.split("filename=")[1].split(";")[0].replaceAll('"', "");
+        } catch {
+          //do nothing
+        }
+      }
+      await useDownloadFileFromResponse(response, "application/json", fileName);
```

Comment:
well if we are using header from the server, then we should also use `Content-Type`?

Reply:
updated to use it